### PR TITLE
(release-4.0.0) hds-2522: clean up announcements

### PR DESCRIPTION
**Pull request by NikoHelle**
## Description

Code cleaning for the announcements. Also the banner's aria-live element was always removed, never announced. Made it visible to screen readers only.

## Related Issue

Closes [HDS-2522](https://helsinkisolutionoffice.atlassian.net/browse/HDS-2522)

## How Has This Been Tested?

Added e2e tests

## Demos:

Links to demos are in the comments

## Add to changelog

- no need

[HDS-2522]: https://helsinkisolutionoffice.atlassian.net/browse/HDS-2522?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/react/src/components/cookieConsentCore/cookieConsentCore.js [hunk 1]
```diff
@@ -408,22 +406,68 @@ export class CookieConsentCore {
     } else {
       window.dispatchEvent(new CustomEvent(cookieEventType.CHANGE, { detail: { acceptedGroups } }));
       if (!this.#settingsPageElement) {
-        this.#announceSettingsSaved();
         this.removeBanner();
+        // removeBanner() removes the setTimeout that shows notification
```
Comment (mrTuomoK) [inline packages/react/src/components/cookieConsentCore/cookieConsentCore.js:410]: commented code?

**Review by mrTuomoK — CHANGES_REQUESTED** [review]
Nothing else to say but the commented code, good work!

File: packages/react/src/components/cookieConsentCore/cookieConsentCore.js (hunk 1 above)
Comment (NikoHelle) [reply packages/react/src/components/cookieConsentCore/cookieConsentCore.js:410]: It's a two line comment to warn that the order of function calls is essential and should never be changed.

**Review by mrTuomoK — APPROVED** [review]
All good now, good job!

**Pull request merged**